.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -34,7 +36,6 @@ Here are all the steps for the release process. Create a new issue at the beginn
 - [ ] [Promote Windows builder worker](https://github.com/habitat-sh/habitat/blob/master/RELEASE.md#promote-the-builder-worker) and confirm build uses new version
 - [ ] Post announcement in [Chef discourse](https://discourse.chef.io/c/habitat)
 - [ ] Post announcement in [Habitat forums](https://forums.habitat.sh/c/announcements)
-- [ ] Tweet announcement from `@habitatsh`
```

Comment:
We should also delete the line in `release.md` that says to do this.

Reply:
@mwrock Good catch; fixed in https://github.com/habitat-sh/habitat/pull/6307/commits/a9741bc2ecda2514362967c1c1a3d66368ef0825